packages/core/src/types/Vector.ts

```diff
@@ -47,6 +47,14 @@ export class Vector2 implements Type {
   public static readonly up = new Vector2(0, 1);
   public static readonly down = new Vector2(0, -1);
 
+  public static readonly middle = new Vector2();
+  public static readonly top = new Vector2(0, -1);
```

Comment:
Could you also add some TSDoc containing the value?
Something like:
```suggestion
  /**
   * A constant for `Vector2(0, -1)`.
   */
  public static readonly top = new Vector2(0, -1);
```
This would make it easier to quickly peak the value in an IDE.